client/destinations.go

```diff
@@ -32,6 +32,19 @@ type SlackAttributes struct {
 	DestinationType string `json:"destination_type"`
 }
 
+type ServiceNowAttributes struct {
+	Name            string `json:"name"`
+	DestinationType string `json:"destination_type"`
+	URL             string `json:"url"`
+	Auth            Auth   `json:"auth"`
+}
+
+type Auth struct {
+	Username string `json:"username"`
+	// Password is only set for requests. Will be empty for API responses
+	Password string `json:"password"`
```

Comment:
if we don't return this field in responses I would expect the diff to complain all the time, what makes this ok?

Reply:
Terraform will use the value saved in its state file to compare to the value in the *.tf file if the field isn't returned by the GET api. I manually tested that this works by applying a terraform file with one of these servicenow destinations then running `terraform plan` and seeing that there are no changes detected.